x-pack/test/functional/apps/ml/anomaly_detection_integrations/single_metric_viewer_dashboard_embeddables.ts

```diff
@@ -83,14 +83,21 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           );
         });
 
-        it('create new single metric viewer panel', async () => {
+        it('should create new single metric viewer panel', async () => {
           await ml.dashboardEmbeddables.clickSingleMetricViewerInitializerConfirmButtonEnabled();
           await PageObjects.timePicker.pauseAutoRefresh();
           await ml.dashboardEmbeddables.assertDashboardPanelExists(testData.panelTitle);
           await ml.singleMetricViewer.assertChartExist();
-          await ml.singleMetricViewer.assertAnomalyMarkerExist();
           await PageObjects.dashboard.saveDashboard(testData.dashboardTitle);
         });
+
+        it('should have anomaly click action menu', async () => {
+          await ml.dashboardEmbeddables.assertDashboardPanelExists(testData.panelTitle);
+          await ml.singleMetricViewer.assertAnomalyMarkerExist();
+          await ml.singleMetricViewer.openAnomalyMarkerActionsPopover();
+          await ml.singleMetricViewer.assertAnomalyActionDiscoverButtonExists();
```

Comment:
Are you using the `ensureAnomalyActionDiscoverButtonClicked` function to test that clicking on the Discover action is opening Discover? I can't see where that new function is being called.

Reply:
Fixed in 80b982d1a1d0def149d03f25c31631daf2b09df1